text/0000-project-groups.md

```diff
@@ -0,0 +1,315 @@
+
+[ffi unwind]: https://github.com/rust-lang/project-ffi-unwind
+[inline asm]: https://github.com/rust-lang/project-inline-asm
+[safe transmute]: https://github.com/rust-lang/project-safe-transmute
+[triage]: https://github.com/rust-lang/team/blob/master/teams/wg-triage.toml
+[rustup]: https://github.com/rust-lang/team/blob/master/teams/wg-rustup.toml
+
+## Project Group Creation
+A project group should have the following;
+
+- **Leads** — At least one person who acts as the leader of the group and is
+  typically responsible for writing the initial charter, handling
+  administrative and communication tasks, as well as delegating those
+  responsibilities to other members in the group.
+- **Liaisons** — A member from a official Rust team that is sponsoring the
+  work, and acts as a point of contact between the team and the group. They
+  may or may be that directly involved, but they should check-in
+  periodically and be able to represent the work in meetings with the team.
+  They should also look out for when this might intersect with other work
+  happening in the team that is beyond the working group itself.
+  - Liaisons may also be but are not required to be one of the leads.
+- **Members** — Individuals who regularly participate and/or
+  contribute to the project group.
+  - Membership requirements for groups are decided by the shepherd and should
+    be stated in the charter.
+  - Initial membership should try to represent people who have already been
+    participating regularly and productively in the respective area.
+  - It is not required for a project group to have a lot of members though,
+    some project groups may only have one or two members including leads and
+    liasions.
+- A charter that defines the scope and intent of the group.
+- A GitHub repository hosted under the `rust-lang` organization containing
+  the charter and instructions for how community members can monitor or
+  participate in the group.
+- Representation on the official [rust-lang.org] website.
+- No _"formal decision making power"_: meaning that they are not able to
+  accept RFCs on `rust-lang/rfcs`.
+  - Groups are of course encouraged to create RFCs as well as advocate their
+    concerns and desired changes to the Rust teams and community.
+- Dedicated space(s) in of Rust's officially managed discussion platforms.
+  - As of the time of this writing this includes [Zulip] and [Discord].
+  - Ideally the group should use the same platform as their parent team to
+    ease communication, though there may be cases where the team agrees to
+    a group trying out a different platform.
+
+### Creating The Charter
+Since project groups are approved by their relevant parent team over the core
+team, it's up to each team decide their specific requirements. However the
+author recommends that a group should try to make a charter that addresses 
+the following questions.
+
+- What value do you see your group bringing to the organisation?
```

Comment:
Hm, I think in one of the refactorings we lost the valuable question: "What support do you need, and separately want, from the Rust organization?" or so.